pkg/parse/parse.go

```diff
@@ -718,6 +718,13 @@ func allowedInVariableName(r rune) bool {
 		r == '-' || r == '_' || r == ':' || r == '~'
 }
 
+func IsValidVariableName(s string) bool {
```

Comment:
Missed this earlier - since this is an exported function, this should have a godoc comment.

Reply:
Done